app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,79 @@
+class MicropostsController < ApplicationController
+  before_action :set_micropost, only: [:show, :edit, :update, :destroy]
+
+  # GET /microposts
+  # GET /microposts.json
+  def index
+    @microposts = Micropost.all
+  end
+
+  # GET /microposts/1
+  # GET /microposts/1.json
+  def show
+  end
+
+  # GET /microposts/new
+  def new
+    @micropost = Micropost.new
+  end
+
+  # GET /microposts/1/edit
+  def edit
+  end
+
+  # POST /microposts
+  # POST /microposts.json
+  def create
+    @micropost = Micropost.new(micropost_params)
+
+    respond_to do |format|
+      if @micropost.save
+        format.html { redirect_to @micropost, notice: 'Micropost was successfully created.' }
+        format.json { render :show, status: :created, location: @micropost }
+      else
+        format.html { render :new }
+        format.json { render json: @micropost.errors,
+                      status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /microposts/1
+  # PATCH/PUT /microposts/1.json
+  def update
+    respond_to do |format|
+      if @micropost.update(micropost_params)
+        format.html { redirect_to @micropost, notice: 'Micropost was successfully updated.' }
+        format.json { render :show, status: :ok, location: @micropost }
+      else
+        format.html { render :edit }
+        format.json { render json: @micropost.errors,
+                      status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # DELETE /microposts/1
+  # DELETE /microposts/1.json
+  def destroy
+    @micropost.destroy
+    respond_to do |format|
+      format.html { redirect_to microposts_url, 
+                    notice: "Micropost was successfully destroyed." }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 62, 69 should be on its own line.